.buildkite/solana-lab.yml

```diff
@@ -0,0 +1,104 @@
+x-default-agents: &default-agents
```

Comment:
Did you mean to put this change into the fecth-google-service-account-key commit?